### ISSUE TITLE
v1.3.0 crashes whenever a player joins the party.

### ISSUE THREAD

**Issue by tamaroth** [C1rdec/Poe-Lurker#55]
It happens every time for me. Below is the contents of the log file:

```
[2020/01/28 23:28:35.184] Trace [1] Lurk with last line (Lurker.ClientLurker)
[2020/01/28 23:30:03.047] Trace [1] Not parsed: 2020/01/28 23:30:02 81715546 ac9 [INFO Client 14588] @To ???????????: I'd like to buy your 5 Exalted Orb for my 815 Chaos Orb in Metamorph. (Lurker.ClientLurker)
[2020/01/28 23:30:39.212] Trace [1] Not parsed: 2020/01/28 23:30:39 81751687 ac9 [INFO Client 14588] @To darksideofshadowz: I'd like to buy your 6 Exalted Orb for my 978 Chaos Orb in Metamorph. (Lurker.ClientLurker)
[2020/01/28 23:31:12.845] Trace [1] Not parsed: 2020/01/28 23:31:12 81785046 ac9 [INFO Client 14588] @To ??????????: I'd like to buy your 6 Exalted Orb for my 978 Chaos Orb in Metamorph. (Lurker.ClientLurker)
[2020/01/28 23:31:39.965] Trace [1] Not parsed: 2020/01/28 23:31:39 81812296 ac9 [INFO Client 14588] @To slimymorph: I'd like to buy your 6 Exalted Orb for my 978 Chaos Orb in Metamorph. (Lurker.ClientLurker)
[2020/01/28 23:32:35.682] Trace [1] Not parsed: 2020/01/28 23:32:35 81868109 1b1 [DEBUG Client 14588] Connect time to instance server was 63ms (Lurker.ClientLurker)
[2020/01/28 23:32:35.682] Trace [1] Not parsed: 2020/01/28 23:32:35 81868031 f5 [INFO Client 14588] Connecting to instance server at 109.95.212.59:6112 (Lurker.ClientLurker)
[2020/01/28 23:32:35.682] Trace [1] Not parsed: 2020/01/28 23:32:35 81868031 d8c [INFO Client 14588] Just before calling client instance session (Lurker.ClientLurker)
[2020/01/28 23:32:35.682] Trace [1] Not parsed: 2020/01/28 23:32:35 81868015 d6f [DEBUG Client 14588] Got Instance Details from login server (Lurker.ClientLurker)
[2020/01/28 23:34:31.591] Trace [1] Not parsed: 2020/01/28 23:34:31 81984000 1b1 [DEBUG Client 14588] Connect time to instance server was 47ms (Lurker.ClientLurker)
[2020/01/28 23:34:31.591] Trace [1] Not parsed: 2020/01/28 23:34:31 81983953 f5 [INFO Client 14588] Connecting to instance server at 172.255.10.236:6112 (Lurker.ClientLurker)
[2020/01/28 23:34:31.591] Trace [1] Not parsed: 2020/01/28 23:34:31 81983953 d8c [INFO Client 14588] Just before calling client instance session (Lurker.ClientLurker)
[2020/01/28 23:34:31.591] Trace [1] Not parsed: 2020/01/28 23:34:31 81983953 d6f [DEBUG Client 14588] Got Instance Details from login server (Lurker.ClientLurker)
[2020/01/28 23:35:13.251] Trace [1] Not parsed: 2020/01/28 23:35:12 82025421 1b1 [DEBUG Client 14588] Connect time to instance server was 31ms (Lurker.ClientLurker)
[2020/01/28 23:35:13.251] Trace [1] Not parsed: 2020/01/28 23:35:12 82025375 f5 [INFO Client 14588] Connecting to instance server at 172.255.8.4:6112 (Lurker.ClientLurker)
[2020/01/28 23:35:13.251] Trace [1] Not parsed: 2020/01/28 23:35:12 82025375 d8c [INFO Client 14588] Just before calling client instance session (Lurker.ClientLurker)
[2020/01/28 23:35:13.251] Trace [1] Not parsed: 2020/01/28 23:35:12 82025375 d6f [DEBUG Client 14588] Got Instance Details from login server (Lurker.ClientLurker)
[2020/01/28 23:36:37.082] Trace [1] Not parsed: 2020/01/28 23:36:36 82109375 1b1 [DEBUG Client 14588] Connect time to instance server was 47ms (Lurker.ClientLurker)
[2020/01/28 23:36:37.082] Trace [1] Not parsed: 2020/01/28 23:36:36 82109312 f5 [INFO Client 14588] Connecting to instance server at 172.255.10.164:6112 (Lurker.ClientLurker)
[2020/01/28 23:36:37.082] Trace [1] Not parsed: 2020/01/28 23:36:36 82109312 d8c [INFO Client 14588] Just before calling client instance session (Lurker.ClientLurker)
[2020/01/28 23:36:37.082] Trace [1] Not parsed: 2020/01/28 23:36:36 82109312 d6f [DEBUG Client 14588] Got Instance Details from login server (Lurker.ClientLurker)
[2020/01/28 23:38:43.060] Trace [1] Not parsed: 2020/01/28 23:38:42 82235359 7c [DEBUG Client 14588] Replacing awake object with id 10052 and type Metadata/Monsters/Spiders/DelveSpiderPacksMedium@83 (Lurker.ClientLurker)
[2020/01/28 23:39:50.382] Trace [1] Not parsed: 2020/01/28 23:39:50 82302812 8e8 [DEBUG Client 14588] Resyncing 16454 (Lurker.ClientLurker)
[2020/01/28 23:40:59.628] Trace [1] Not parsed: 2020/01/28 23:40:59 82371968 1b1 [DEBUG Client 14588] Connect time to instance server was 47ms (Lurker.ClientLurker)
[2020/01/28 23:40:59.628] Trace [1] Not parsed: 2020/01/28 23:40:59 82371906 f5 [INFO Client 14588] Connecting to instance server at 172.255.8.20:6112 (Lurker.ClientLurker)
[2020/01/28 23:40:59.628] Trace [1] Not parsed: 2020/01/28 23:40:59 82371906 d8c [INFO Client 14588] Just before calling client instance session (Lurker.ClientLurker)
[2020/01/28 23:40:59.628] Trace [1] Not parsed: 2020/01/28 23:40:59 82371906 d6f [DEBUG Client 14588] Got Instance Details from login server (Lurker.ClientLurker)
[2020/01/28 23:41:41.423] Trace [1] Not parsed: 2020/01/28 23:41:41 82413859 ac9 [INFO Client 14588] @To namcreierLULW: give me a sec, please (Lurker.ClientLurker)
[2020/01/28 23:41:41.423] Trace [1] Not parsed: 2020/01/28 23:41:41 82413765 8e8 [DEBUG Client 14588] Resyncing 11515 (Lurker.ClientLurker)
[2020/01/28 23:41:41.423] Trace [1] Not parsed: 2020/01/28 23:41:41 82413765 8e8 [DEBUG Client 14588] Resyncing 11513 (Lurker.ClientLurker)
[2020/01/28 23:41:41.423] Trace [1] Not parsed: 2020/01/28 23:41:41 82413765 8e8 [DEBUG Client 14588] Resyncing 11511 (Lurker.ClientLurker)
[2020/01/28 23:41:52.473] Trace [1] Not parsed: 2020/01/28 23:41:52 82424921 ac9 [INFO Client 14588] @To namcreierLULW: delve (Lurker.ClientLurker)
[2020/01/28 23:42:39.178] Trace [1] Not parsed: flags: 129 (Lurker.ClientLurker)
[2020/01/28 23:42:39.178] Trace [1] Not parsed: target_id: 13815 (Lurker.ClientLurker)
[2020/01/28 23:42:39.178] Trace [1] Not parsed: object_id: 11502 (Lurker.ClientLurker)
[2020/01/28 23:42:39.178] Trace [1] Not parsed: action_id: 44884 (Lurker.ClientLurker)
[2020/01/28 23:42:39.178] Trace [1] Not parsed: skill_instance_id: 32777 (Lurker.ClientLurker)
[2020/01/28 23:42:39.178] Trace [1] Not parsed: 2020/01/28 23:42:38 82471375 1e3 [WARN Client 14588] Instant/Triggered actionwas serialized to the client, but the client failed to execute the action. (Lurker.ClientLurker)
[2020/01/28 23:42:39.178] Trace [1] Not parsed: 2020/01/28 23:42:38 82471375 262 [WARN Client 14588] Client couldn't execute an instant/triggered action from the server (Lurker.ClientLurker)
[2020/01/28 23:42:53.214] Trace [1] Not parsed: 2020/01/28 23:42:53 82485687 1b1 [DEBUG Client 14588] Connect time to instance server was 31ms (Lurker.ClientLurker)
[2020/01/28 23:42:53.214] Trace [1] Not parsed: 2020/01/28 23:42:53 82485625 f5 [INFO Client 14588] Connecting to instance server at 172.255.10.236:6112 (Lurker.ClientLurker)
[2020/01/28 23:42:53.214] Trace [1] Not parsed: 2020/01/28 23:42:53 82485625 d8c [INFO Client 14588] Just before calling client instance session (Lurker.ClientLurker)
[2020/01/28 23:42:53.214] Trace [1] Not parsed: 2020/01/28 23:42:53 82485625 d6f [DEBUG Client 14588] Got Instance Details from login server (Lurker.ClientLurker)
[2020/01/28 23:42:56.368] Error [1] Exception has been thrown by the target of an invocation.
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at Lurker.UI.ViewModels.TradebarViewModel.AddActiveOffer(OfferViewModel offer)
   at Lurker.UI.Models.TradebarContext.AddToActiveOffer(OfferViewModel offer)
   at Lurker.UI.ViewModels.OfferViewModel.Invite()
   at Lurker.UI.ViewModels.OfferViewModel.MainAction()
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Caliburn.Micro.ActionMessage.<>c.<.cctor>b__33_0(ActionExecutionContext context)
   at Caliburn.Micro.ActionMessage.Invoke(Object eventArgs)
   at System.Windows.Interactivity.TriggerBase.InvokeActions(Object parameter)
   at System.Windows.Interactivity.EventTriggerBase.OnEvent(EventArgs eventArgs)
   at System.Windows.Interactivity.EventTriggerBase.OnEventImpl(Object sender, EventArgs eventArgs)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseEvent(RoutedEventArgs e)
   at System.Windows.Controls.Primitives.ButtonBase.OnClick()
   at System.Windows.Controls.Button.OnClick()
   at System.Windows.Controls.Primitives.ButtonBase.OnMouseLeftButtonUp(MouseButtonEventArgs e)
   at System.Windows.UIElement.OnMouseLeftButtonUpThunk(Object sender, MouseButtonEventArgs e)
   at System.Windows.Input.MouseButtonEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   at System.Windows.UIElement.OnMouseUpThunk(Object sender, MouseButtonEventArgs e)
   at System.Windows.Input.MouseButtonEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.UIElement.RaiseEvent(RoutedEventArgs args, Boolean trusted)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at Lurker.UI.App.Main()Inner exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at Lurker.UI.ViewModels.TradebarViewModel.AddActiveOffer(OfferViewModel offer)
   at Lurker.UI.Models.TradebarContext.AddToActiveOffer(OfferViewModel offer)
   at Lurker.UI.ViewModels.OfferViewModel.Invite()
   at Lurker.UI.ViewModels.OfferViewModel.MainAction() (Lurker.UI.AppBootstrapper)
[2020/01/28 23:45:59.899] Trace [1] Lurk with last line (Lurker.ClientLurker)
```

If more info is needed, don't hesitate to ask.

**Comment by C1rdec** [comment]
I'm not able to reproduce the issue...

**Comment by tamaroth** [comment]
When someone who wants to buy my item joins my party. I will try to debug this tomorrow evening, thankfully you've included pdb, so it shouldn't be that much of a problem.

**Comment by C1rdec** [comment]
Let me know when you have some free time to investigate on discord

**Comment by tamaroth** [comment]
I'm available now, for approximately next 4 to 5 hours.

I have tried building PoE-Lurker on my own, unfortunately assembly `Lurker.Items` could not be found. Is it internal reference or can it be downloaded from somewhere official?

**Comment by C1rdec** [comment]
You are not using the installer ?

**Comment by C1rdec** [comment]
For the Lurker.Items I need to make a nuget package

**Comment by tamaroth** [comment]
I am, for the release version. But I wanted to build Lurker from the sources so that I can debug the issue more easily and perhaps contribute in the future..

**Comment by C1rdec** [comment]
Yeah I'll make a nuget tonight

**Comment by tamaroth** [comment]
Ah, makes sense. If you're interested I'm available on PoE's discord under the same as here.

**Comment by tamaroth** [comment]
Misclicked, sorry :)

**Comment by C1rdec** [comment]
But I'm working right now...

**Comment by tamaroth** [comment]
Oh right, I'm sorry. I forgot things like timezones exist. I have managed to locate and fix the bug in #56

**Comment by C1rdec** [comment]
Thx

**Comment by C1rdec** [comment]
Merged into master will make a 1.3.1
